cms/src/main/resources/api-specification/codegen-templates/index.mustache

```diff
@@ -41,74 +41,58 @@
                                             <p>{{{markdown notes levelToNormaliseHeadingsTo=4}}}</p>
 
                                             {{#isAnyTrue this hasPathParams hasHeaderParams hasBodyParam hasFormParams hasQueryParams}}
-                                                <h4>Request</h4>
+                                                <h4 class="nhsd-t-heading-l">Request</h4>
```

Comment:
i believe the guidance is for h4's to use the class `nhsd-t-heading-m`?

Reply:
Indeed, good point!

Now fixed, together with a bunch of other `<hX>` elements in the template.